operator/controllers/lifecycle/keptnevaluation/controller.go

```diff
@@ -264,10 +265,21 @@ func (r *KeptnEvaluationReconciler) fetchDefinitionAndProvider(ctx context.Conte
 	if err := r.Client.Get(ctx, namespacedDefinition, evaluationDefinition); err != nil {
 		return nil, nil, err
 	}
+
+	if evaluationDefinition.Spec.Source == providers.KeptnMetricProviderName {
```

Comment:
shouldn't we ship the operator with a keptn metrics provider definition already added to the namespace at installation?

Reply:
I am not fan of this.... it will only introduce garbage resources to the cluster